src/commands/auth/jwt/grant.ts

```diff
@@ -5,73 +5,95 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-import * as os from 'os';
-import { flags, FlagsConfig, SfdxCommand } from '@salesforce/command';
-import { AuthFields, AuthInfo, AuthRemover, Messages, SfError } from '@salesforce/core';
+import { Flags, loglevel } from '@salesforce/sf-plugins-core';
+import { AuthFields, AuthInfo, AuthRemover, Logger, Messages, SfError } from '@salesforce/core';
 import { get, getString, Optional } from '@salesforce/ts-types';
-import { Prompts } from '../../../prompts';
+import { Interfaces } from '@oclif/core';
+import { AuthBaseCommand } from '../../../authBaseCommand';
 import { Common } from '../../../common';
 
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/plugin-auth', 'jwt.grant');
 const commonMessages = Messages.loadMessages('@salesforce/plugin-auth', 'messages');
 
-export default class Grant extends SfdxCommand {
+export default class Grant extends AuthBaseCommand<AuthFields> {
+  public static readonly summary = messages.getMessage('summary');
   public static readonly description = messages.getMessage('description');
-  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+  public static readonly examples = messages.getMessages('examples');
   public static aliases = ['force:auth:jwt:grant'];
 
-  public static readonly flagsConfig: FlagsConfig = {
-    username: flags.string({
-      char: 'u',
-      description: messages.getMessage('username'),
+  public static readonly flags = {
+    'user-name': Flags.string({
```

Comment:
Is this something we are doing in other username flags? We use the word `username` most of our docs.

Same question about the `o` short-flag . We reserved that for `--target-org`, I know both refer to an org username but haven't seen it being re-used in other flags.

Reply:
I'll change flag name to `username` but will leave short flag as `-u`. There is follow-on work scheduled to reconcile this plugin with others, so will address short flag names then.